packages/ogmios/ogmios-v6.1.0.yaml

```diff
@@ -16,7 +16,7 @@ installSteps:
         - --node-socket
         - /ipc/node.socket
         - --node-config
-        - /config/mainnet/cardano-node/config.json
+        - /config/preprod/cardano-node/config.json
```

Comment:
Don't hardcode the network. There's a magic var for this: `{{ .Context.Network }}`